src/modules/DOM.js

```diff
@@ -1,40 +1,81 @@
 import Countries from './rest_countries.js';
+import Involvement from './involvement.js';
+import PopUp from './popUp.js';
 
 class DOM {
   constructor() {
     this.countries = new Countries();
     this.container = document.querySelector('.wrapper');
+    this.involvement = new Involvement();
+    this.popUp = new PopUp();
   }
 
-  createHtmlContent = async (country) => {
-    const countryInfo = this.countries.getCountryShortInfo(country);
-    const html = `
-    <article class="wrapper__card">
-      <img class="wrapper__card__img" src="${
-        (await countryInfo).flagURl
-      }" alt="${(await countryInfo).name}" />
-      <div class="wrapper__card__details">
-        <h3 class="wrapper__card__details__title">${
-          (await countryInfo).name
-        }</h3>
-        <div class="wrapper__card__details__likes">
-          <button class="wrapper__card__details__likes-btn"><i class="bi bi-heart-fill"></i></button>
-          <p></p>
-        </div>
-      </div>
-      <button class="wrapper__card__comments-btn card__btn">Comments</button>
-    </article>
-    `;
-    return html;
+  createCardEl = async (country) => {
+    const countryInfo = await this.countries.getCountryShortInfo(country);
+    const { name, flagURl } = countryInfo;
+    let likes = await this.involvement.getLikeAmount(name);
+
+    // The element structure
+    const _ = document;
+    const article = _.createElement('article');
+    article.className = 'wrapper__card';
+
+    const img = _.createElement('img');
+    img.className = 'wrapper__card__img';
+    img.src = flagURl;
+    img.alt = name;
+
+    const cardDetails = _.createElement('div');
+    cardDetails.className = 'wrapper__card__details';
+
+    const h3 = _.createElement('h3');
+    h3.className = 'wrapper__card__details__title';
+    h3.innerText = name;
+
+    const likeDiv = _.createElement('div');
+    likeDiv.className = 'wrapper__card__details__likes';
+
+    const btnLike = _.createElement('button');
+    btnLike.className = 'wrapper__card__details__likes-btn';
+    btnLike.innerHTML = '<i class="bi bi-heart-fill"></i>';
+
+    const likeAmount = _.createElement('p');
+    likeAmount.innerText = `${likes} like(s)`;
+
+    const btnComments = _.createElement('button');
+    btnComments.innerText = 'Comment';
+    btnComments.className = 'wrapper__card__comments-btn card__btn';
+
+    likeDiv.appendChild(btnLike);
+    likeDiv.appendChild(likeAmount);
+
+    cardDetails.appendChild(h3);
+    cardDetails.appendChild(likeDiv);
+
+    article.appendChild(img);
+    article.appendChild(cardDetails);
+    article.appendChild(btnComments);
+
+    // ActionListeners
+    btnLike.addEventListener('click', async () => {
+      likes += 1;
+      likeAmount.innerText = `${likes} like(s)`;
+      await this.involvement.addLike(name);
+    });
```

Comment:
The implementation of the likes button and counter is good. However, I recommend keeping the heart-icon red if the button is clicked, and deleting the like if it is unenabled. For this, you could use the localStorage, to avoid the same user generating a lot of likes. This is OPTIONAL, but it could be better for the UX.